packages/types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shapeshiftoss/types",
-  "version": "4.3.2",
+  "version": "5.0.0",
```

Comment:
Do we need this?

Reply:
semantic release will overwrite this when it does the version bump. It's needed so that the yarn install at the beginning of the build won't symlink the package for the other packages that still want the 4.x version.

If we don't update it like this then the build will fail.

Reply:
This ^ it gives us the green dot in CI so we can check that nothing _inside_ the types package broke.